### PR TITLE
Handle htest objects in get/find_statistic

### PR DESCRIPTION
**Pull request by bwiernik**
Closes https://github.com/easystats/insight/issues/492

I put the `data.name` object in "Parameter" and the name of the test statistic in the "statistic" attribute

### CODE REVIEW

**Review by strengejacke — APPROVED** [review]

**Review by IndrajeetPatil — APPROVED** [review]

File: R/get_statistic.R [hunk 1]
```diff
@@ -76,6 +76,32 @@ get_statistic.default <- function(x, column_index = 3, verbose = TRUE, ...) {
   out
 }
 
+#' @export
+get_statistic.htest <- function(x, ...) {
+
+  if (x$method == "Fisher's Exact Test for Count Data") {
+    out <- data.frame(
+      Parameter = x$data.name,
+      Statistic = x$p.value,
```
Comment (IndrajeetPatil) [inline R/get_statistic.R:85]: Is this right? Why don't we call this column `p`?

For what it's worth, we don't have a `Statistic` column for this test in `parameters()` output.

So maybe we can also return `NA_real_`?

``` r
library(parameters)

TeaTasting <- matrix(c(3, 1, 1, 3),
  nrow = 2,
  dimnames = list(
    Guess = c("Milk", "Tea"),
    Truth = c("Milk", "Tea")
  )
)

m <- fisher.test(TeaTasting, alternative = "greater")
parameters(m)
#> Fisher's Exact Test for Count Data
#> 
#> Odds.Ratio | CI_low |     p
#> ---------------------------
#> 6.41       |   0.31 | 0.243
#> 
#> Alternative hypothesis: true odds ratio is greater than 1
```

<sup>Created on 2022-01-24 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>

Comment (bwiernik) [reply R/get_statistic.R:85]: My thinking is that for this test, the test statistic and p value are the same thing. It's a weird edge case, but seemed like the best option versus NA. We don't have a function for returning p values from model objects. Should we include p and perhaps parameter values for htest?

**Comment by strengejacke** [comment]
I slightly improved the printing stuff here:

``` r
library(parameters)

TeaTasting <- matrix(c(3, 1, 1, 3),
                     nrow = 2,
                     dimnames = list(
                       Guess = c("Milk", "Tea"),
                       Truth = c("Milk", "Tea")
                     )
)

m <- fisher.test(TeaTasting, alternative = "greater")
parameters(m)
#> Fisher's Exact Test for Count Data
#> 
#> Odds Ratio |      95% CI |     p
#> --------------------------------
#> 6.41       | [0.31, Inf] | 0.243
#> 
#> Alternative hypothesis: true odds ratio is greater than 1
```

<sup>Created on 2022-01-27 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by strengejacke** [comment]
Can we merge this?

File: R/get_statistic.R (hunk 1 above)
Comment (IndrajeetPatil) [reply R/get_statistic.R:85]: @strengejacke What's your opinion on this? After this comment is resolved, we can merge this PR.
